tests/test_time_frequency.py

```diff
@@ -357,6 +357,79 @@ def test_A_weighting(min_db):
         assert not np.any(a_range < min_db)
 
 
+@pytest.mark.parametrize("min_db", [None, -40, -80])
```

Comment:
These tests should not have prints in them

Reply:
oops my bad!